pkg/expression/builtin_other.go

```diff
@@ -1052,7 +1052,11 @@ func (b *builtinGetRealVarSig) evalReal(ctx EvalContext, row chunk.Row) (float64
 	}
 	varName = strings.ToLower(varName)
 	if v, ok := sessionVars.GetUserVarVal(varName); ok {
-		return v.GetFloat64(), false, nil
```

Comment:
What about the implementation`(b *builtinGetRealVarSig) vecEvalReal`

Reply:
+1

Reply:
+1

Reply:
Yes, please. Much appreciated.

Let's talk about enabling vectorization for get/set var function in the future.